content/sensu-core/2.0/reference/silencing.md

```diff
@@ -0,0 +1,193 @@
+subscription is not provided, it will be substituted with a wildcard (asterisk):
+`$SUBSCRIPTION:*` or `*:$CHECK`.
+
+### Attributes
+check        | 
+-------------|------ 
+description  | The name of the check the entry should match 
+required     | true, unless `subscription` is provided
+type         | String
+example      | {{< highlight shell >}}"check": "haproxy_status"{{</ highlight >}}
+example      | {{< highlight shell >}}"check": "haproxy_status", "subscription" : "load_balancer"{{</ highlight >}}
+
+subscription | 
+-------------|------ 
+description  | The name of the subscription the entry should match 
+required     | true, unless `check` is provided
+type         | String
+example      | {{< highlight shell >}}"subscription": "entity:i-424242"{{</highlight>}}
+example      | {{< highlight shell >}}
+  "subscription": "entity:i-424242",
+  "check" : "haproxy_status"
+{{</ highlight >}}
+
+id           | 
+-------------|------ 
+description  | Silencing identifier generated from the combination of a subsription name and check name. 
+required     | false - this value cannot be modified 
+type         | String
+example      | {{< highlight shell >}}
+  "id": "appserver:mysql_status",
+  "check": "mysql_status",
+  "subscription": "appserver",
+{{</ highlight >}}
+
+begin        | 
+-------------|------ 
+description  | Time at which silence entry goes into effect, in epoch. 
+required     | false 
+type         | Integer 
+example      | {{< highlight shell >}}
+  "begin": 1512512023,
+  "check": "disk_utilization",
+  "subscription": "entity:i-424242"
+{{</ highlight >}}
+
+expire       | 
+-------------|------ 
+description  | Number of seconds until this entry should be deleted. 
+required     | false 
+type         | Integer 
+default      | -1
+example      | {{< highlight shell >}}
+  "expire": 3600,
+  "check": "disk_utilization",
+  "subscription": "entity:i-424242"
+{{</ highlight >}}
+
+expire_on_resolve       | 
+-------------|------ 
+description  | If the entry should be deleted when a check begins return OK status (resolves). 
+required     | false 
+type         | Boolean 
+default      | false 
+example      | {{< highlight shell >}}
+  "expire_on_resolve": true,
+  "check": "disk_utilization",
+{{</ highlight >}}
+
+
+creator      | 
+-------------|------ 
+description  | Person/application/entity responsible for creating the entry. 
+required     | false 
+type         | String 
+default      | null 
+example      | {{< highlight shell >}}
+  "creator": "Application Deploy Tool 5.0",
+  "subsription": "appservers",
+  "check": "app_status"
+{{</ highlight >}}
+
+reason       | 
+-------------|------ 
+description  | Explanation for the creation of this entry.
+required     | false 
+type         | String 
+default      | null 
+example      | {{< highlight shell >}}
+  "creator": "mercedes",
+  "subsription": "webserver",
+  "check": "app_status",
+  "reason": "rebooting the world"
+{{</ highlight >}}
+
```

Comment:
I'm just realizing now that we should probably also include the `environment` & `organization` attributes and I totally forgot to do that for filters!

Reply:
Ah yeah, good point. I will add them here and to mutators as I don't think I did so either!

Reply:
@palourde added org/env attributes. Can you double check the wording? Figure we can copy-pasta to other reference docs and keep it consistent :)

Reply:
@mercul3s Maybe capitalize `Sensu`?

And about the default value, I believe it will use the current env/org from `sensuctl`, at least with the flags/interactive modes. I'm wondering if we should mark those as required in the docs 🤔

Reply:
OK, I can make it more clear that it uses the current org/env from sensuctl. I think we document them as required and further describe that behavior when we get to the RBAC docs (and maybe also for setting up sensuctl), but not go into too much detail here. They are optional as long as the cli has been configured.

Reply:
👍 👍 👍